### DISCUSSION TITLE
Trigger Automation (Reminder) 1 hour prior to game time.

### DISCUSSION THREAD

**Discussion by bcayton** [vasqued2/ha-teamtracker#118] [Q&A]
Help! I'm trying to figure out how to trigger an HA automation to remind me one hour before game time of the game. I can't see to get the trigger correct. I found documentation somewhere that the Attribute was "Kickoff in" and then "TO" was in 1 hour, however I can't seem to find that documentation again. Can anyone point me in the right direction?

**Comment by vasqued2** [comment]
My guess is the TO value should be "in an hour" but it could be "in 1 hour" or "in a hour".  I don't have anything starting within an hour to know for sure but you should be able to look at the value of the `kickoff_in` attribute around an hour before it starts and see what it says.

**Reply by bcayton** [reply]
Thanks David, anyway to get it to announce with TV network the game will be on? I have tried {{state_attr('sensor.team_tracker.attributes', 'Tv network') }} but it always returns "none".

**Reply by vasqued2** [reply]
If the API returns a network, it will be in that attribute.  If it doesn't return one, the value is set to none and there is nothing I can do.
